Gemfile.lock

```diff
@@ -161,13 +160,12 @@ GEM
       net-protocol
       timeout
     nio4r (2.5.8)
-    nokogiri (1.13.3)
-      mini_portile2 (~> 2.8.0)
+    nokogiri (1.13.3-x86_64-linux)
```

Comment:
@felixclack this is peculiar to me, the `-linux` suffix

Reply:
I'll check it locally for me

Reply:
It generated a diff when I ran this locally. I've added a commit to also add the Mac version here.

Reply:
Nice!